package.json

```diff
@@ -64,7 +64,7 @@
     "gulp-concat": "^2.6.1",
     "html-webpack-plugin": "4.0.0-beta.5",
     "htmllint": "^0.8.0",
-    "htmllint-cli": "0.0.7",
+    "htmllint-cli": "github:kevinlee12/htmllint-cli",
```

Comment:
Shouldn't this include a version number?

Reply:
Agreed, but since I don't want to push a new version of htmllint-cli, I've included the commit instead.